containers/layouts/LayoutsSection.js

```diff
@@ -110,6 +118,23 @@ const LayoutsSection = () => {
                     <ViewModeRadios viewMode={ViewMode} onChange={handleChangeViewMode} loading={loadingViewMode} />
                 </Field>
             </Row>
+            <Row>
```

Comment:
You need to hide this row if conversation grouping is off.

Reply:
Ouch, completely forgot about that!

Reply:
Ok, it's done, but the code for `LayoutsSection` has become a bit less uniform since I created a new component, `ConversationGrouping` to handle this effect.

Reply:
I removed the `ConversationGrouping` component thanks to your suggestion below, and I think it looks better now